.github/workflows/cd/helm.yaml

```diff
@@ -0,0 +1,52 @@
+name: cd/helm
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'helm/**'
+  workflow_dispatch:
+    inputs:
+      OVERRIDE_EXISTING:
+        type: boolean
+        default: false
+        required: false
+
+jobs:
+  package:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: azure/setup-helm@v1
+      - run: |
+          helm package -d helm helm/charts/*
+      - uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-region: ${{ secrets.AWS_REGION }}
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      - run: |
+          for chart in *.tgz; do
+            if aws s3 ls s3://${{ env.HELM_BUCKET }}/$chart && ! ${{ inputs.OVERRIDE_EXISTING }} ; then
+              echo "Skipping $chart which already exists."
+              continue
+            fi
+
+            echo "Syncing $chart to ${{ env.HELM_BUCKET }}."
+            mkdir -p ${chart%%.tgz}
+            cp -a $chart ${chart%%.tgz}
```

Comment:
This is a hack. `helm repo index` requires a directory so if there are multiple charts in a directory, everything will be indexed.